### ISSUE TITLE
Edit value of output keys in SOAR actions for better discovery

### ISSUE THREAD

**Issue by tzturner** [techservicesillinois/secops-soar-tdx#96]
In #90 it was discovered that a newly-created ticket's ID can be printed out once the `create_ticket` action is complete by using a `debug` block in a SOAR playbook and plugging in [the value of the `data_path` key in the action's output.](https://github.com/techservicesillinois/secops-soar-tdx/blob/main/src/app/app.json#L217-L222)

```json
"output": [
                {
                    "data_path": "action_result.data.*.id",
                    "data_type": "string"
                }
          ],
```

Playbook:

![image](https://github.com/techservicesillinois/secops-soar-tdx/assets/25675046/170ed311-470a-485e-80fe-85329cb5d608)

Notice that this value ends in `id` instead of `ticket_id` or something similar.

A request has been made (at lower priority) to edit this value to make it more discoverable for someone searching for a ticket's ID.

## Tasks

- [x] verify the test suite passes
- [x] rename the  "data_path": "action_result.data.*.id", to something like  "data_path": "action_result.data.*.tdx_ticket_id",
- [x] verify the test suite passes
- [x] verify the new name in SOAR
